Gemfile.lock

```diff
@@ -44,7 +44,7 @@ GEM
       jsonapi-renderer (>= 0.1.1.beta1, < 0.3)
     active_storage_base64 (1.0.0)
       rails (~> 6.0)
-    activeadmin (2.6.1)
+    activeadmin (2.7.0)
```

Comment:
### [activeadmin](https://activeadmin.info)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 2.6.1 → 2.7.0

[_~~change-log~~, [source-code](http://github.com/activeadmin/activeadmin)_]





<details>
<summary>Commits</summary>

A change of **101** commits. See the full changes on [the compare page](https://github.com/activeadmin/activeadmin/compare/v2.6.1...v2.7.0).

These are the first 10 commits:
- (5a98857) [Move `npm_push` task inside `ReleaseManager` (#6078)](https://github.com/activeadmin/activeadmin/commit/5a988578abb6eff4a71113913704e7756b17caac)
- (230d52d) [Bump draper from 3.1.0 to 4.0.0 in /gemfiles/rails_52 (#6082…](https://github.com/activeadmin/activeadmin/commit/230d52d29b4347f4a3ddd211949b5a019a3f53cc)
- (63478d9) [Bump draper from 3.1.0 to 4.0.0 in /gemfiles/rails_60_turbol…](https://github.com/activeadmin/activeadmin/commit/63478d9008913c61dd07c0eeb8eb8730ec2bd871)
- (b93db38) [Bump draper from 3.1.0 to 4.0.0 (#6084)](https://github.com/activeadmin/activeadmin/commit/b93db38aaf4c2ef09ae60dddd4aa47a766d2914c)
- (83dcaaa) [Bump rubocop-rspec from 1.37.1 to 1.38.0 (#6088)](https://github.com/activeadmin/activeadmin/commit/83dcaaaf93ab620991a923876e61a788f11ba66c)
- (1290efa) [Bug in compare URL (#6085)](https://github.com/activeadmin/activeadmin/commit/1290efa1fc7984badebe774f108d886a1e82624c)
- (eacde20) [Add flashes wrapper to logged_out view (#6086)](https://github.com/activeadmin/activeadmin/commit/eacde2041d212d4f4deb452fa805fb07e6219e7c)
- (ab3e21c) [Bump rollup from 1.31.0 to 1.31.1 (#6093)](https://github.com/activeadmin/activeadmin/commit/ab3e21c38d5489ca281b0a0da0b5b96be2a84a14)
- (a92ee4d) [Bump rubocop-rspec from 1.38.0 to 1.38.1 (#6094)](https://github.com/activeadmin/activeadmin/commit/a92ee4d2394f3f5b8ee15de7f26e4ce754834842)
- (e805537) [Bump database_cleaner from 1.8.2 to 1.8.3 in /gemfiles/rails…](https://github.com/activeadmin/activeadmin/commit/e805537e2a79734ae6cf3566067fc552989ca1fc)


</details>